Documentation/install/aws/aws-terraform.md

```diff
@@ -111,6 +111,17 @@ $ cp examples/terraform.tfvars.aws build/${CLUSTER}/terraform.tfvars
 
 Edit the parameters with your AWS details, domain name, license, etc. [View all of the AWS specific options][aws-vars] and [the common Tectonic variables][vars].
 
+### Set Console login secrets
+
+Before deploying, set the user name and password for the initial Console log in. Set these sensitive values in the environment, from where Terraform will encrypt them:
```

Comment:
Terraform will encrypt these sensitive values from the environment in which they are set.